content/blogs/gui-setup-part2-partitioning.md

```diff
@@ -0,0 +1,88 @@
+---
+title:      "1st-stage GUI setup - October-November 2023: Making partitioning UI work"
+author:     "hbelusca"
+date:       2024-01-11
+tags:       [ newsletter, gui-setup ]
+---
+
+Greetings!
+
+As you may have noticed, I have been quite silent about my work, not writing on a regular basis blog entries about what I have done so far.
+Well, I am more concerned about getting _actual_ coding done and working before writing about it, instead of writing about half-done not-yet-tested code, much like what the GUI setup code was during these previous months.
+
+This is why it is only now that I'm presenting, in this second blog post, the work I made during the months of October and November, in preparation for the new 1st-stage ReactOS GUI setup.
+
+During the first week of October I finished addressing some last issues that arose with the partial setupapi winesync [I started back in September](/blogs/gui-setup-part1-setupapi).
+
+I then finally started the main part of this project: working on the GUI setup proper.
+The main necessary work for it resides in three functionalities:
+- disk partitioning and bootloader installation,
+- file copying,
+- settings saving in registry.
+While the last two features were already partly implemented (most of the code being shared with the existing text-mode installer ``USETUP'') and will be discussed in future blog posts, the first feature: disk partitioning, necessitated more work than I originally anticipated. Indeed, this functionality is shared with USETUP, but is used in a slightly different manner in the GUI setup.
```

Comment:
```suggestion
  - While the last two features were already partly implemented (most of the code being shared with the existing text-mode installer `USETUP`) and will be discussed in future blog posts, the first feature: disk partitioning, necessitated more work than I originally anticipated. Indeed, this functionality is shared with USETUP, but is used in a slightly different manner in the GUI setup.
```

Reply:
Added this paragraph instead in its own separate line.